org.mixedrealitytoolkit.uxcomponents/Editor/CreateElementMenus.cs

```diff
@@ -235,10 +251,74 @@ private static void CreateSimpleActionButton(MenuCommand menuCommand)
             Undo.SetCurrentGroupName("Create SimpleActionButton");
 
             GameObject simpleActionButton = CreateElementFromPath(SimpleActionButtonPath, menuCommand);
-
             Undo.RecordObject(simpleActionButton, "Added SimpleActionButton instance.");
         }
 
+        #region Experimental Buttons with dynamic Frontplate
+        [MenuItem("GameObject/UI/MRTK/Dynamic Frontplate (Experimental)/Empty Button w dynamic Frontplate (Experimental)")]
+        private static void CreateEmptyButtonDynamicFrontplate(MenuCommand menuCommand)
```

Comment:
Let's not add a menu for the empty button....I don't think that's very useful given there is no visual component.

Reply:
Removed in latest commit.  Resolving.